src/test/java/edu/hm/hafner/analysis/parser/VeraCodePipelineScannerParserTest.java

```diff
@@ -20,38 +20,57 @@ class VeraCodePipelineScannerParserTest extends StructuredFileParserTest {
 
     @Override
     protected void assertThatIssuesArePresent(final Report report, final SoftAssertions softly) {
-        softly.assertThat(report).hasSize(5);
+        softly.assertThat(report).hasSize(7);
 
         softly.assertThat(report.get(0))
                 .hasSeverity(Severity.WARNING_HIGH)
                 .hasMessage("org.slf4j.Logger.info")
-                .hasFileName("com/sample/LoggingFilter.java")
+                .hasFileName("src/main/java/com/sample/LoggingFilter.java")
+                .hasPackageName("com.sample")
                 .hasType("Improper Output Neutralization for Logs")
                 .hasLineStart(28);
         softly.assertThat(report.get(1))
                 .hasSeverity(Severity.WARNING_NORMAL)
                 .hasMessage("set")
                 .hasFileName("react/dist/esm/data.js")
+                .hasPackageName("-")
                 .hasType("Use of Hard-coded Password")
                 .hasLineStart(25);
         softly.assertThat(report.get(2))
                 .hasSeverity(Severity.WARNING_LOW)
                 .hasMessage("management:endpoint:health:show-details:")
                 .hasFileName("BOOT-INF/classes/application.yml")
+                .hasPackageName("application")
                 .hasType("Information Exposure Through Sent Data")
                 .hasLineStart(1);
         softly.assertThat(report.get(3))
                 .hasSeverity(Severity.WARNING_LOW)
                 .hasMessage("nosourcefile")
                 .hasFileName("-")
+                .hasPackageName("-")
                 .hasType("No source_file present")
                 .hasLineStart(0);
         softly.assertThat(report.get(4))
                 .hasSeverity(Severity.WARNING_LOW)
                 .hasMessage("nofiles")
                 .hasFileName("-")
+                .hasPackageName("-")
                 .hasType("No files present")
                 .hasLineStart(0);
+        softly.assertThat(report.get(5))
+                .hasSeverity(Severity.WARNING_HIGH)
+                .hasMessage("child_process.spawn")
+                .hasFileName("lib/optimizer/Optimizer.js")
+                .hasPackageName("-")
+                .hasType("Improper Neutralization of Special Elements used in an OS Command ('OS Command Injection')")
+                .hasLineStart(24);
+        softly.assertThat(report.get(6))
+                .hasSeverity(Severity.WARNING_HIGH)
+                .hasMessage("org.slf4j.Logger.info")
+                .hasFileName("src/main/java/com/othersample/LoggingFilter.java")
+                .hasPackageName("com.othersample")
+                .hasType("Improper Output Neutralization for Logs")
+                .hasLineStart(55);
```

Comment:
I did not see this in my first review (and I do not use the tool on my own), but typically the type is an ID and the message is a human readable text. It seems that you did switch those values in your parser. Was this intended? Please note that the tab selections and the column width preferences assume that type is a small text and message a long one.

So from the file I would use the mapping:

`title`: `type`
`issue_type`: `message`
`issue_type_id`: `category`

Reply:
Good spot. I did not realize that the type should be short text.

I mapped `issue_type` to `type` because then in UI it is easier to filter issues for `SQL Injection` or `Improper Output Neutralization for Logs`. 

The `issue_type_id` is quite ambiguous, so I did not map it.

Reply:
Fixed now.

Reply:
You can see the live instance at https://ci.jenkins.io/job/Plugins/job/analysis-model/job/master/1451/spotbugs/

The message is the title of the expandable area (in bold), this title is included in the search filter. But it is only visible if you expand the row. If that is not what you want then you can map `issue_type` to something else. This depends somehow how you typically want to interact with the issues.